src/Components/TaskReview/QuizReview.js

```diff
@@ -0,0 +1,144 @@
+import React from 'react';
+import { useState } from 'react';
+import './QuizReview.css';
+import { GridList, GridListTile } from '@material-ui/core';
+
+export default function QuizReview(props) {
+    console.log(props.qa);
+    const perc = Math.floor((props.pointsAwarded/props.pointsTotal) * 100 );
+    const percentageBarWidth = 320;
+    const [qNum, setQNum] = useState(1); 
+
+    function percentageBarLeftStyle() {
+        return {
+            'width' : ((perc/100) * percentageBarWidth).toString() + 'px',
+            'height': '30px',
+            'backgroundColor' : '#EA6868',
+            'marginLeft' : '180px',
+            'borderBottomLeftRadius' : '12px',
+            'borderTopLeftRadius' : '12px',
+            'paddingTop' : '7px'
+        };
+    }
+
+    function percentageBarRightStyle() {
+        return {
+            'width' : (((100 - perc)/100) * percentageBarWidth).toString() + 'px',
+            'backgroundColor' : '#E0E0E0',
+            'borderBottomRightRadius' : '12px',
+            'borderTopRightRadius' : '12px'
+        };
+    }
+
+    const optionGrayStyle = {
+        'backgroundColor' : '#F2F2F2',
+        'color' : '#4F4F4F',
+        'fontFamily' : 'Poppins',
+        'fontStyle' : 'normal',
+        'fontWeight': 'normal',
+        'fontSize' : '16px',
+        'width' : '570px',
+        'height' : '18px',
+        'padding' : '7px',
+        'borderRadius' : '25px',
+    };
+
+    const optionGreenStyle = {
+        'backgroundColor' : '#27AE60',
+        'color' : 'white',
+        'fontFamily' : 'Poppins',
+        'fontStyle' : 'normal',
+        'fontWeight': 'normal',
+        'fontSize' : '16px',
+        'width' : '570px',
+        'height' : '18px',
+        'padding' : '7px',
+        'borderRadius' : '25px',
+    };
+
+    const optionRedStyle = {
+        'backgroundColor' : '#EB5757',
+        'color' : 'white',
+        'fontFamily' : 'Poppins',
+        'fontStyle' : 'normal',
+        'fontWeight': 'normal',
+        'fontSize' : '16px',
+        'width' : '570px',
+        'height' : '18px',
+        'padding' : '7px',
+        'borderRadius' : '25px',
+    }
+
+    function getOptionsStyle(option, selected, correctOption) {
+        if((option === selected) && (selected !== correctOption)) {
+            return optionRedStyle;
+        }
+        if(option === selected) {
+            return optionGreenStyle;
+        }
+        if(option === correctOption) {
+            return optionGreenStyle;
+        }
+        return optionGrayStyle;
```

Comment:
Good job rendering the different question colors for correct/incorrect. Looks really good